app/templates/components/hyper-table.hbs

```diff
@@ -119,11 +119,8 @@
 
   {{#if (eq _collection.length 0)}}
     <div class="hypertable__state hypertable__state--empty">
-      {{#if (has-block "inverse")}}
         {{yield to="inverse"}}
-      {{else}}
-        No Data
-      {{/if}}
```

Comment:
why this change ? 🤔

Reply:
My bad shouldn't have committed that, mock data doesn't show otherwise